parquet-hadoop/src/main/java/org/apache/parquet/hadoop/BloomFilterDataReader.java

```diff
@@ -36,20 +37,22 @@
   private final ParquetFileReader reader;
   private final Map<String, ColumnChunkMetaData> columns;
   private final Map<String, BloomFilter> cache = new HashMap<>();
+
   public BloomFilterDataReader(ParquetFileReader fileReader, BlockMetaData block) {
     this.reader = fileReader;
     this.columns = new HashMap<>();
     for (ColumnChunkMetaData column : block.getColumns()) {
       columns.put(column.getPath().toDotString(), column);
     }
   }
+
   @Override
   public BloomFilter readBloomFilter(ColumnDescriptor descriptor) {
     String dotPath = Strings.join(descriptor.getPath(), ".");
     ColumnChunkMetaData column = columns.get(dotPath);
     if (column == null) {
       throw new ParquetDecodingException(
-        "Cannot load Bloom filter data, unknown column: " + dotPath);
+        "Cannot read Bloom filter data from meta data of column: " + dotPath);
```

Comment:
I know, it is not about the current change but I think, it is wrong to throw an exception in any case we cannot read the bloom filter. Filtering shall not be crucial for reading the data. If a filtering cannot work we shall simply not do the filtering and read all the data.
Moreover, it might be valid that the writer did not write a bloom filter for the related column. It is a decision to be made by the writers.
If the bloom filter reading leads the a sever I/O problem than still, the reading will get one anyway or we are able to successfully read the data without filtering.
It might be fine to throw an exception at reading but we have to swallow these exceptions at some point (and properly log them) to not fail the file reading only because we cannot read the bloom filter.

Reply:
Make sense. I will catch exception in filter logic and print error.